### PR TITLE
Add warning about free hosting being discontinued when CSMM runs in donor-only mode

### PR DESCRIPTION
**Pull request by niekcandaele**

### CODE REVIEW

File: views/layouts/applicationLayout.sejs [hunk 1]
```diff
@@ -195,6 +195,13 @@
     <div id="alert-socket-disconnect" class="alert alert-warning" role="alert">
       <strong>Warning: lost connection. Please try refreshing this page or check your internet connection.</strong>
     </div>
+
+    <% if(process.env.CSMM_DONOR_ONLY === 'true'){ %>
```
Comment (halkeye) [inline views/layouts/applicationLayout.sejs:199]: eww, env inside template 😭

File: views/layouts/applicationLayout.sejs [hunk 2]
```diff
@@ -195,6 +195,13 @@
...
+      <div class="alert alert-warning" role="alert">
+        <strong>(!) Free hosting is being discontinued - join <a href="https://catalysm.net/discord">the Discord</a> for more information</strong>
```
Comment (halkeye) [inline views/layouts/applicationLayout.sejs:201]: Again, can you make sure the only way to get info isn't just discord? I'm sure some don't want to sign up.

File: views/layouts/applicationLayout.sejs (hunk 1 above)
Comment (niekcandaele) [reply views/layouts/applicationLayout.sejs:199]: ¯\_(ツ)_/¯

File: views/layouts/applicationLayout.sejs (hunk 2 above)
Comment (niekcandaele) [reply views/layouts/applicationLayout.sejs:201]: Ohh I didnt realize that's what you meant. Should we add a full explanation to the docs then? Probably related: https://github.com/CatalysmsServerManager/docs/pull/246

**Pull request merged**